### ISSUE TITLE
there is no obvious effect when change compressImageQuality value on Android

### ISSUE THREAD

**Issue by PatWu16** [ivpusic/react-native-image-crop-picker#556]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.13.1
- react-native v0.43.3

### Platform
Tell us to which platform this issue is related

- Android

### Expected behaviour

compressImageQuality property can be useful

### Actual behaviour

original picture size: 2.7M
when has compressImageQuality property, set 0 - 1, get picture size is 72643 B, there is no obvious effect when change compressImageQuality value; when hasn't compressImageQuality property, get picture size is 2.7M;

### Attachments

// stacktrace or any other useful debug info

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by ivpusic** [comment]
do you have some time to investigate this?

**Comment by PatWu16** [comment]
@ivpusic  Haven't you met this situation before?

**Comment by PatWu16** [comment]
Now that I've met this problem, I'll try to find out the reason and solve it. if anyone has a solution, please share it.

**Comment by PatWu16** [comment]
@ivpusic After investigation,when hasn't compressImageQuality property, get original picture size;when set   
 compressImageQuality = 1,picture  will be compressed 0.1 of original size

**Comment by profiVideos** [comment]
Just thought I'd pipe in here.  For me it's working on Android without a problem.  I did extensive testing of the compressImageQuality on Android today.  Tested at least 15 variations and determined 0.25 is a nice trade-off of quality to file size.  Using the latest React Native 0.52 & the latest image-crop-picker (installed it today).  File Sizes for an Image (original was 1920x1080 pixels) down to 1056x768 pixels @ 1.0 (100%) of 481K down to 25K at 0.25.

**Comment by fubar** [comment]
@profiVideos This may be worth putting into the readme as I'd assume most people who implement this lib wonder what compression value to pick.

**Comment by PatWu16** [comment]
yes，it's no problem less than @1.0,  i think our need is between @1.0 and not setting compressImageQuality.
The final solution was that doesn't set compressImageQuality, and use another plugin "react-native-image-resizer".
i hope that author can adjust the compression value to allow the users to choose a larger range

**Comment by profiVideos** [comment]
That is of course your choice. However there is nothing here to fix. Think of the range as being 0 to 100 % with 1 being equivalent  to 100 %.  It just a different scale. I did extensive testing and found it woks as expected.  Don't believe me. Try using a compression ratio of 0.08 or roughly 8% and you'll see an image with real banding. As  I wrote before, I'm happy with at setting of. 2 to. 25.

On Feb 2, 2018 01:54, PatWu <notifications@github.com> wrote:yes，it's no problem less than @1.0,  i think our need is between @1.0 and not setting compressImageQuality.
The final solution was that doesn't set compressImageQuality, and use another plugin "react-native-image-resizer".
i hope that author can adjust the compression value to allow the users to choose a larger range
